docs/user_guide/ttl.md

```diff
@@ -14,10 +14,32 @@ store-version, after the push finishes, but before swapping the read traffic ove
 replay phase where the last N seconds of buffered real-time data is replayed. The replayed data gets overlaid on top of
 the data from the Full Push.
 
-This can be leveraged in order to achieve a TTL behavior. By scheduling periodic empty pushes and configuring how far 
-back to replay the real-time data (N), one can control the TTL parameters. The time to live is defined by N, which acts 
-as a "minimum TTL", while the "maximum TTL" is N + delay between each empty push. For example, if you schedule a daily 
-empty push, and N = 6 days, then the oldest data in your store will be at least 6 days old, and at most 7 days old.
+This can be leveraged in order to achieve a TTL behavior. Currently, there are two options that leverages this mechanism. 
+
+### Empty push
+By scheduling periodic empty pushes and configuring how far back to replay the real-time data (N), one can control the 
+TTL parameters. The time to live is defined by N, which acts as a "minimum TTL", while the "maximum TTL" is N + delay 
+between each empty push. For example, if you schedule a daily empty push, and N = 6 days, then the oldest data in your 
+store will be at least 6 days old, and at most 7 days old.
+
+The N is as known as the store-level rewind time, that's a part of hybrid store configuration.
```

Comment:
Can we rephrase this sentence? Some suggestions:

1. Cut out the "as known as".
2. We might want to explicitly refer to `hybrid-rewind-seconds` being the relevant admin tool parameter to use for configuring this.